packages/client/src/runtime/RequestHandler.ts

```diff
@@ -59,15 +62,6 @@ export type HandleErrorParams = {
   transaction?: PrismaPromiseTransaction
 }
 
-export type Request = {
```

Comment:
We had to add more and more things from `RequestParams` here, so it made less and less sense to keep this object to me. I've decided we could just pass `RequestParams` to dataloader directly instead.

Reply:
I agree, better this way.